ls1intum/Artemis
# `Programming exercises`: Do not collapse feedback if there is only one feedback group

### Checklist
#### General



#### Client

### Motivation and Context


When there is only one category available, the Feedback group should not hidden by default. This is a simple usability feature that saves students a single click

### Description

Adds a check for the amount of feedback groups. If there is only one group, set that groups open property to true

### Steps for Testing

Prerequisites:
- a student
- instructor
- an exercise e.g. (modeling, file submission are easy to test) 

1. Create a submission for the exercise 
2. Grade it afterwards as instructor - Only give positive feedback
3. When you check the feedback as student, the group content should be shown (see screenshot below).

### Screenshots


Default before:
![image](https://user-images.githubusercontent.com/25489463/219857750-e1e74c9a-6473-480d-87cb-572cbc66e400.png)

Default now:
![image](https://user-images.githubusercontent.com/25489463/219857773-bd5ecd4c-0b66-455f-9546-29640f960988.png)
